css/styles.css

```diff
@@ -12,24 +12,22 @@ h1 {
   color: white;
 }
 
-label,
-input {
-  font-size: 12px;
+#edit-profile-title {
+  font-weight: bold;
 }
 
-#bio {
-  width: 100%;
+label,
+input {
+  font-size: 16px;
 }
 
-.avatar-photo {
-  width: 300px;
+.btn-primary {
+  background-color: rgb(67, 150, 71);
 }
```

Comment:
Don't use classes that interfere with bootstrap's classes, just go ahead and make your own utility class, you can also just use their class and overwrite their styling with a second class you created on your own. Your styling will have a higher specificity, since it gets loaded in second